src/js/view/controls/components/timeslider.js

```diff
@@ -106,6 +106,12 @@ class TimeSlider extends Slider {
             .change('buffer', this.onBuffer, this)
             .change('streamType', this.onStreamType, this);
 
+        // Clear cues on player model's playlistItem change event
```

Comment:
I was just going to do `this._model._model`, but it looked ugly and lint complained about dangling `_`. I can disable linter for this line if that is preferred

Reply:
Why are you not removing `.change('playlistItem', this.onPlaylistItem, this)`?